### ISSUE TITLE
ffqueue has bug when merge multiple files

### ISSUE THREAD

**Issue by adonais** [bswebdk/FFQueue#8]
open Tools->Merge multiple sources to one..., 
if the source file path contains non-ascii characters, command execution failed.
i guess  the problem affects all platforms.
```html
--- a/src/utils/FFQToolJobs.cpp	2019-02-28 03:28:16.000000000 +0800
+++ b/src/utils/FFQToolJobs.cpp	2019-03-29 07:45:57.000000000 +0800
@@ -769,8 +769,8 @@
 
     }
 
-    //Save the list as ascii
-    wxScopedCharBuffer scb = s.ToAscii();
+    //Save the list as utf-8
+    wxScopedCharBuffer scb = s.ToUTF8();
     bool ok = lf->Write((void*)scb.data(), scb.length());
     lf->Close();
     delete lf;
```

**Comment by bswebdk** [comment]
Thank you for the hint, it will be fixed in the next release!

**Comment by bswebdk** [comment]
Hopefully resolved in 1.7.53
